mmpose/datasets/datasets/bottom_up/bottom_up_base_dataset.py

```diff
@@ -53,12 +53,12 @@ def __init__(self,
         self.ann_info['inference_channel'] = data_cfg['inference_channel']
         self.ann_info['dataset_channel'] = data_cfg['dataset_channel']
 
-        self.db = []
+        self.image_set_index = []
```

Comment:
how to understand `image set index`? `image set`'s index? image `set index`? what is an `index`?
why is this name better than `db`?

Reply:
change the name to `image_index`